src/components/usa-details/index.js

```diff
@@ -0,0 +1,63 @@
+import { LitElement, html, css, unsafeCSS, nothing } from "lit";
+import { classMap } from "lit/directives/class-map.js";
+import styles from "./usa-details.css.js";
+import uswdsCoreStyles from "@uswds/uswds/scss/uswds-core?inline";
+
+/**
+ * @summary The usa-details component.
+ *
+ * @slot details-body - Content for the details panel
+ *
+ * @attribute {Boolean} open - Set the panel to be open on load
+ * @attribute {String} name - Add a group name if the element is part of a details group
+ *
+ * @cssprop --theme-details-font-family - Sets the font family for the details element
+ * @cssprop --theme-details-border-color - Sets the border width for the details element
+ * @cssprop --theme-details-border-width - Sets the border color for the details element
+ * @cssprop --theme-details-background-color - Sets the background color of the content panels
+ * @cssprop --theme-details-summary-background-color - Sets the background color of the summary element
+ *
+ * @tagname usa-details
+ */
+export class UsaDetails extends LitElement {
+  static styles = [unsafeCSS(uswdsCoreStyles), styles];
+
+  static properties = {
+    bordered: { type: Boolean }
+  }
```

Comment:
**Question (non-blocking):** Should `open` be listed here?

Reply:
I was only listing the props that should be added to `<usa-details>` here. In the current iteration, the `open` attribute should be added to the the child `<details>` element, so I left it off the properties list for now. Let me know if that should change!